advanced/Templates/pihole-FTL.service

```
@@ -9,8 +9,39 @@
 # Description:       Enable service provided by pihole-FTL daemon
 ### END INIT INFO
 
+# Get PID of main pihole-FTL process
+FTLCONFFILE="/etc/pihole/pihole-FTL.conf"
+DEFAULT_PID_FILE="/run/pihole-FTL.pid"
```

Initialise PID variable as empty string to assure it cannot be passed accidentially:
```suggestion
DEFAULT_PID_FILE="/run/pihole-FTL.pid"
FTL_PID=''
```